### PR TITLE
New section added in frontpage

### PR DESCRIPTION
**Pull request by jyotiattri**
This adds the functionality for [**website - Add section to display the 6 camel projects with logos**](https://issues.apache.org/jira/projects/CAMEL/issues/CAMEL-14772?filter=allopenissues)

### CODE REVIEW

**Comment by jyotiattri** [comment]
Hi @zregvart 
Is there anything that I need to do to fix the build errors?

File: antora-ui-camel/src/css/frontpage.css [hunk 1]
```diff
@@ -237,6 +237,10 @@ section.frontpage h1 {
     flex: 100%;
   }
 
+  .box-new {
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Can't we use the existing `box` class? Naming things `-new` leads to naming the next new thing `-newer` and the thing after that one `-newest` and so on. This is never a good practice. Read [this](https://www.w3.org/QA/Tips/goodclassnames).

File: antora-ui-camel/src/css/frontpage.css [hunk 2]
```diff
@@ -273,3 +277,48 @@ section.frontpage h1 {
     text-align: center;
   }
 }
+
+section.frontpage.projects-new {
+  display: flex;
+  display: -webkit-box;
+  display: -ms-flexbox;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: We don't add browser prefixes, we rely only on standards.

File: antora-ui-camel/src/css/frontpage.css [hunk 3]
```diff
@@ -273,3 +277,48 @@ section.frontpage h1 {
...
+  flex-wrap: wrap;
+}
+
+.box-new {
+  -webkit-box-flex: 33%;
+  -ms-flex: 33%;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Remove browser prefixes

File: antora-ui-camel/src/css/frontpage.css [hunk 4]
```diff
@@ -273,3 +277,48 @@ section.frontpage h1 {
...
+section.frontpage .box-new p {
+  font-style: italic;
+  text-align: center;
+}
+
+section.frontpage.projects .project h2 {
+  white-space: nowrap;
+  padding: 1rem 0;
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Remove browser prefixes.

**Comment by zregvart** [comment]
Preview is [here](https://builds.apache.org/job/Camel.website/job/PR-270/Preview/index.html). This also needs a rebase.

File: antora-ui-camel/src/css/frontpage.css (hunk 1 above)
Comment (jyotiattri) [reply antora-ui-camel/src/css/frontpage.css]: This PR is regarding [website - Add section to display the 6 camel projects with logos](https://issues.apache.org/jira/browse/CAMEL-14772).
I created the icons and the design according to the tweet mentioned the issue.
I need help with SVG icons. Can you please help me with SVG icons?

**Comment by aashnajena** [comment]
Are the preview builds somehow broken? I see a very distorted page when I open this link on chrome. There was some inconsistency with my [PR-207](https://github.com/apache/camel-website/pull/207) also.


preview

**Comment by jyotiattri** [comment]
@aashnajena, I am not sure why it is like that for you, it is working fine for me. I tested in both Firefox and Chrome.

File: antora-ui-camel/src/css/frontpage.css (hunk 1 above)
Comment (zregvart) [reply antora-ui-camel/src/css/frontpage.css]: We have Apache Camel logo [here](https://github.com/apache/camel/tree/master/docs/img) not sure how Claus created that image in the tweet.

**Comment by jyotiattri** [comment]
In this, I have made changes in CSS and Add svg images into it.

File: content/_index.md [hunk 5]
```diff
@@ -53,55 +53,67 @@ Camel supports around 50 data formats, allowing to <mark>translate messages</mar
...
-
-<a class="significant" href="./manual/latest/">Read the docs</a>
-<a class="significant" href="./components/latest/eips/enterprise-integration-patterns.html">EIP reference</a>
-<a class="significant" href="./components/latest/">Component reference</a>
-{{< /div >}}
-
-{{< div "project" >}}
-# Camel K
+{{< section "frontpage camelProjects" >}}
+# Apache Camel 3 - Projects
```
Comment (zregvart) [inline content/_index.md:57]:
```suggestion
# Platforms Camel runs on
```

File: antora-ui-camel/src/css/frontpage.css [hunk 6]
```diff
@@ -246,7 +246,8 @@ section.frontpage h2 {
 }
 
 @media screen and (max-width: 626px) {
-  .split {
+  .split,
+  .projectDetails {
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css:250]: projectDetails -> platforms

File: antora-ui-camel/src/css/frontpage.css [hunk 7]
```diff
@@ -290,3 +291,37 @@ section.frontpage h2 {
...
+
+section.frontpage .projectDetails h2 {
+  text-align: center;
+  text-transform: capitalize;
+  width: 100%;
+  color: var(--heading-font-color);
+}
+
+section.frontpage .projectDetails p {
+  font-style: italic;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css:325]: Remove italic

**Comment by zregvart** [comment]
Camel logo [we already](https://github.com/apache/camel-website/blob/master/antora-ui-camel/src/img/logo-d.svg) have on the website (look at the footer)

Here are optimized versions of other logos you can use [logos.zip](https://github.com/apache/camel-website/files/4420786/logos.zip)

File: antora-ui-camel/src/css/frontpage.css [hunk 8]
```diff
@@ -290,3 +291,63 @@ section.frontpage h2 {
     text-align: center;
   }
 }
+
+section.frontpage.camelProjects {
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css:295]: camelProjects -> platforms

File: antora-ui-camel/src/css/frontpage.css [hunk 9]
```diff
@@ -290,3 +291,63 @@ section.frontpage h2 {
...
+  position: relative;
+}
+
+.other-half-circle {
+  position: absolute;
+  width: 250px;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+}
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css:353]: All these need to be scoped to `.frontpage .platforms`

Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css:353]: I don't see this used.

**Comment by jyotiattri** [comment]
@zregvart, OK I will do all changes. Thanks

**Comment by zregvart** [comment]
@jyotiattri I've cleaned this up in #337, let's continue there.

**Comment by jyotiattri** [comment]
@zregvart, Ok Sure.

**Pull request closed without merging**